frontend/src/features/mapSideBar/tabs/propertyDetails/update/components/LandMeasurementTable.tsx

```diff
@@ -0,0 +1,197 @@
+        const newSqFeet = convertArea(value, AreaUnitTypes.Acres, AreaUnitTypes.SquareFeet);
+        const newHa = convertArea(value, AreaUnitTypes.Acres, AreaUnitTypes.Hectares);
+
+        return {
+          [AreaUnitTypes.SquareMeters]: newSqMeters,
+          [AreaUnitTypes.SquareFeet]: newSqFeet,
+          [AreaUnitTypes.Hectares]: newHa,
+          [AreaUnitTypes.Acres]: value,
+        };
+      });
+    }
+  }, [focus, acres]);
+
+  const handleInputChange = useCallback(
+    (newValue: number, areaUnitTypeCode: string) => {
+      setFocus(areaUnitTypeCode);
+      setState(prevState => ({ ...prevState, [areaUnitTypeCode]: newValue }));
+      if (typeof onChange === 'function') {
+        onChange(newValue, areaUnitTypeCode);
+      }
+    },
+    [onChange],
+  );
+
+  return (
+    <>
+      <TableCaption>Land measurement</TableCaption>
+      <StyledTable role="table" className="table">
+        <div role="rowgroup" className="tbody">
+          <div className="tr-wrapper">
+            <div role="row" className="tr">
+              <div role="cell" title="" className="td right">
+                <input
+                  type="number"
+                  step=".01"
+                  value={sqMeters}
+                  onChange={e =>
+                    handleInputChange(e.target.valueAsNumber, AreaUnitTypes.SquareMeters)
+                  }
+                />
+              </div>
+              <div role="cell" title="" className="td left">
+                sq. metres
+              </div>
+            </div>
+          </div>
+          <div className="tr-wrapper">
+            <div role="row" className="tr">
+              <div role="cell" title="" className="td right">
+                <input
+                  type="number"
+                  step=".01"
+                  value={sqFeet}
+                  onChange={e =>
+                    handleInputChange(e.target.valueAsNumber, AreaUnitTypes.SquareFeet)
+                  }
+                />
+              </div>
+              <div role="cell" title="" className="td left">
+                sq. feet
+              </div>
+            </div>
+          </div>
+          <div className="tr-wrapper">
+            <div role="row" className="tr">
+              <div role="cell" title="" className="td right">
+                <input
+                  type="number"
+                  step=".01"
+                  value={ha}
+                  onChange={e => handleInputChange(e.target.valueAsNumber, AreaUnitTypes.Hectares)}
+                />
+              </div>
+              <div role="cell" title="" className="td left">
+                hectares
+              </div>
+            </div>
+          </div>
+          <div className="tr-wrapper">
+            <div role="row" className="tr">
+              <div role="cell" title="" className="td right">
+                <input
```

Comment:
Just want to put a reminder here that you can override the contents of a cell with a custom component (such as an input).
`{
    Header: 'Summary',
    accessor: 'summary',
    align: 'left',
    clickable: true,
    sortable: true,
    width: 80,
    maxWidth: 120,
    Cell: (props: CellProps<IContactSearchResult>) => {
      const { hasClaim } = useKeycloakWrapper();
      if (hasClaim(Claims.CONTACT_VIEW)) {
        return <Link to={`/contact/${props.row.original.id}`}>{props.row.original.summary}</Link>;
      }
      return props.row.original.summary;
    },
  },`

Reply:
Not sure if you recall that I mentioned a re-rendering issue with that approach. I tried using a table and putting the input like you show above but the whole table was re-rendering on each keypress so the input would loose focus (as it is a new rendered component) and that didn't work.

So I had to "re-create" the look and feel of a table manually. It works because `Table.scss` exists and the styles are global. If/when Table gets converted to styled components we would loose those styles but I felt that I didn't need to duplicate the css for now